src/app.ts

```diff
@@ -0,0 +1,11 @@
+import express from 'express';
+import { routes } from './controllers/routes';
+
+const app = express();
+const port = 3000;
+
+app.use('/', routes);
+
+app.listen(port, () => {
```

Comment:
1. export `app` from this file
2. in `index.ts`, start the actual server by importing `app` and doing `app.listen()`. This allows you to import the app in your test framework for E2E testing as well.